arch/Kconfig.v2

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# ToDo: handle oot archs
```

Comment:
Is this something to add support for in the initial PR or later on? Since hwmv1 will still be available, seems like a future PR is okay for this

Reply:
I would say acceptable for merging to collab branch but should be addressed before merging collab branch to main.

Reply:
oot arch now handled.